### ISSUE TITLE
HTTP Error 401

### ISSUE THREAD

**Issue by TDGalea** [CarlEdman/godaddy-ddns#14]
Hello!
I've just tried implementing this, but regardless of how I run the script (whether with direct arguments or a %file) Python reports HTTP error 401. I've tried recreating my key & secret but still getting the same thing. Any ideas?

Python3 version 3.6.8. As far as I can tell, all dependencies are installed (although apt doesn't find any of them - perhaps I'm just being dumb.

**Comment by soneups** [comment]
TDGalea - have you already created the DNS entry that you are trying to update via this script? - I have this working to host a DNS name (ISP Dynamic IP) for a local openvpn server so may be able to offer some help.

**Comment by TDGalea** [comment]
Hey soneups. After posting this, I ended up just figuring out what the domain and format of the API is and created my own Bash script which is now working flawlessly. I don't intend to advertise my own work here so I won't go further into that. As such I'm not desperate for this to work now, but I would still like to help.

The DNS entry does exist, and I tried multiple API keys but got the same result.

**Comment by soneups** [comment]
So I call the script explicity via 'python3 -path-to/godaddy_dns.py %/path-to/godaddy-ddns.config'
and in the config file i have these four lines
domain
--key
key
--secret
secret
but i had already created the domain - hth

**Comment by TDGalea** [comment]
Yep, pretty much did the same thing but got the 401 error every single time.

**Comment by soneups** [comment]
and the same 401 error if you just parse it straight to the script using just the command line switches/variables? I know i had some initial issues getting up and running but it was so long ago now and it 'just' works now...
